### DISCUSSION TITLE
Should Ctrl+Click work with `useMenuItem`?

### DISCUSSION THREAD

**Discussion by JoaoSilvaGrafana** [adobe/react-spectrum#3062] [General]
Hi everyone 👋 ,

We are using `react-spectrum` to help us to build an accessible navigation, so we use several of your hooks. Recently a user raised an issue that they expected a Ctrl+Click on a menu item to open in a new tab, however it doesn't seem to be possible with `useMenuItem`, because that hijacks the click and just calls `onAction` where we don't have any contextual information of how it was called.

I wanted to start a discussion on this to see if it is something that you would want to consider adding or if there is some work around to this?

Thanks!

**Comment by snowystinger** [comment]
Just to confirm, this is specifically when Menu items are links? If so, this is probably best represented by this issue https://github.com/adobe/react-spectrum/issues/1244

**Reply by JoaoSilvaGrafana** [reply]
Yes that is what I meant! Thank you for pointing me to that issue, someone mentioned recently the same issue with cmd+click, so I will just keep an eye on that 👍
